### PR TITLE
Js feature

### PR DESCRIPTION
**Pull request by Dachrono**
Add function to the nav button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,22 +7,20 @@
...
-        </nav>  
+        <nav id="navBar" class="NavBar"> 
+            <a href="#cards">Portfolio</a>
+            <a href="#About">About</a> 
+            <a href="#contForm">Contact</a>  
+        </nav>
+        
+        <div id="menu" class="menu">
+            <a href="javascript:void(0)" onclick="hello()" class="fa fa-bars"></a>
+        </div>
```
Comment (koachcode) [inline index.html]: - On clicking the Hamburger menu, it shows the mobile menu, however, the hamburger icon did not change to an `X` which show signify a close button, Please can you change the Hamburger menu to an `X` when the mobile menu is open and back to the `Ham burger menu ` when the mobile menu is closed as in the Figma design. 

##### Your design 
image

##### Figma Design 
image

File: index.html [hunk 2]
```diff
@@ -7,22 +7,20 @@
...
-            </div>
-            
-            <div class="menu">
-                <a href="" class="fa fa-bars"></a>
-            </div>     
-        </nav>  
+        <nav id="navBar" class="NavBar"> 
+            <a href="#cards">Portfolio</a>
+            <a href="#About">About</a> 
+            <a href="#contForm">Contact</a>
```
Comment (koachcode) [inline index.html]: - From the project requirements which state that
     - `When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.` 
     - `When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.`
  
Please make these requirements present in your portfolio functionality.

File: index.html [hunk 3]
```diff
@@ -7,22 +7,20 @@
...
-            </div>     
-        </nav>  
+        <nav id="navBar" class="NavBar"> 
+            <a href="#cards">Portfolio</a>
+            <a href="#About">About</a> 
+            <a href="#contForm">Contact</a>  
+        </nav>
+        
+        <div id="menu" class="menu">
+            <a href="javascript:void(0)" onclick="hello()" class="fa fa-bars"></a>
```
Comment (koachcode) [inline index.html]: - I noticed when i click on the hamburger menu, it has a background color, please kindly take off the hover effect as that's what gives it the color, and try to follow the Figma design, so you have a similar-looking design as much as possible. 

##### Your design 
image

##### Figm design 
image

File: JS/script.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-unused-vars
+function hello() {
```
Comment (koachcode) [inline JS/script.js]: - [Optional] Please give a function name that describes what the function does, eg `toggleMobileMenu` as `hello` is not descriptive.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @Dachrono, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter error :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please Check figma design to get the actual look of the mobile menu


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Dachrono) [reply index.html]: Solved

File: index.html (hunk 2 above)
Comment (Dachrono) [reply index.html]: Solved

File: index.html (hunk 3 above)
Comment (Dachrono) [reply index.html]: Solved

File: JS/script.js (hunk 4 above)
Comment (Dachrono) [reply JS/script.js]: Solved

**Review by DuaneDave — APPROVED** [review]
Hi @Dachrono 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good job navigating a user to specific section on clicking the nav links

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**